crates/nu-command/src/filters/select.rs

```diff
@@ -214,18 +214,15 @@ fn select(
                         Vec::new(),
                     ));
                 }
-                if unique_rows.contains(val) {
-                    return Err(ShellError::GenericError(
-                        "Select can't get the same row twice".into(),
-                        "duplicated row index".into(),
-                        Some(*span),
-                        None,
-                        Vec::new(),
-                    ));
+                if !unique_rows.contains(val) {
+                    unique_rows.insert(*val);
+                }
```

Comment:
With `unique_rows` being a `HashSet` this check doesn't make much sense.

Reply:
You're absolutely right!
I totally missed that :sweat_smile: 
Thanks for the review!
i removed it